src/nvim/api/vim.c

```diff
@@ -1306,14 +1309,14 @@ Array nvim_get_api_info(uint64_t channel_id)
 ///     are thus ignored. Clients must only use keys defined in this or later
 ///     versions of nvim!
 ///
-/// @param attributes Informal attributes describing the client. Clients might
-///                   define their own keys, but the following are suggested:
-///     - "website" Website of client (for instance github repository)
-///     - "license" Informal description of the license, such as "Apache 2",
-///                 "GPLv3" or "MIT"
-///     - "logo"    URI or path to image, preferably small logo or icon.
-///                 .png or .svg format is preferred.
+/// @param attributes Arbitrary string:string map of informal client properties.
```

Comment:
Restricting this to string-only values. https://github.com/neovim/go-client/pull/36#issuecomment-476418658